boards/arm/stm32f746g_disco/stm32f746g_disco.yaml

```diff
@@ -6,7 +6,7 @@ toolchain:
   - zephyr
   - gnuarmemb
   - xtools
-ram: 256
+ram: 8192
```

Comment:
The way it's wired up on the board means that only half of it is accessible. The reason I added it was because it was done that way on another board that I took my inspiration from.

https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/mimxrt1064_evk/mimxrt1064_evk.yaml

Reply:
this is going to break test case filtering which considers the amount of RAM available.
This just seems horribly wrong to me. Can't we put the actual amount of RAM in kilobytes here? what is preventing this?